README.md

```diff
@@ -77,6 +78,9 @@ format>
 
 Where:
 
+--simple-warmup
+Do not perform the simple warmup call.
```

Comment:
We should also update the comment next to the flag. Should now say something like:

Perform the simple warmup call.

While we are here, do we also want to expand this a bit to explain what simple warmup call is?

Reply:
whoops, done in bb70db154224cb50d06d315eb7e3359b25a8be9f (thanks!)